packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -270,17 +270,24 @@ to learn more about AWS Lambda's X-Ray support.
 
 ## Lambda with Profiling
 
+The following code configures the lambda function with CodeGuru profiling. By default, this creates a new CodeGuru
+profiling group -
+
 ```ts
 import * as lambda from '@aws-cdk/aws-lambda';
 
 const fn = new lambda.Function(this, 'MyFunction', {
-    runtime: lambda.Runtime.NODEJS_10_X,
+    runtime: lambda.Runtime.PYTHON_3_6,
     handler: 'index.handler',
-    code: lambda.Code.fromInline('exports.handler = function(event, ctx, cb) { return cb(null, "hi"); }'),
+    code: lambda.Code.fromAsset('lambda-handler'),
     profiling: true
 });
 ```
 
+The `profilingGroup` property can be used to configure an existing CodeGuru profiler group.
+
+CodeGuru profiling is supported for all Java runtimes and Python3.6+ runtimes.
```

Comment:
Could we instead link to CodeGuru Profiler documentation for this information, so it's less likely to get out of date? 

For example, this page: https://docs.aws.amazon.com/codeguru/latest/profiler-ug/what-is-codeguru-profiler.html

Reply:
Maybe a direct link to "What languages are supported by CodeGuru Profiler?" would be nicer? 

URL: https://docs.aws.amazon.com/codeguru/latest/profiler-ug/what-is-codeguru-profiler.html#what-is-language-support